### PR TITLE
doc: add core notes 2021-04-09

### PR DESCRIPTION
**Pull request by gemarcano**
### Pull Request Overview

Adds meeting notes for 2021-04-09.

### Testing Strategy

N/A

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x]  Ran `make prepush`.

### CODE REVIEW

File: doc/wg/core/notes/core-notes-2021-04-09.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ - Johnathan Van Why
+ - Vadim Sukhomlinov
+ - Branden Ghena
+ - Philip Levis
+
+## Updates
+
+ - Brad: Working on error standarization. Hopefully at the end should be understandable. Fairly invasive changes.
+ - Pat: Student taking a fresh stab at making a lora capsule. Undergrad asking if there's anything they can do. Dry-run at hardware CI server. Supposedly the students already have RPis. Minimum viable product is to get stuff from github, flash program, verify GPIO is toggled and report that back to CI.
+ - Amit: PhD student, working on progress security, based on previous lightweight context research. Use case, when unwrapped a key for an endpoint (plaintext encryption for private key), switch into a context with limited access to memory and narrow interface to back out to return results, and limited access to hardware, hopefully can make a stronger argument to limit the leaking of the private key across contexts. Vadim talked about some similar work/implementation/use case leveraging drivers controlling access to specific flash pages and some other interfances.
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2021-04-09.md:19]:
```suggestion
 - Amit: PhD student, working on process security, based on previous lightweight context research. Use case, when unwrapped a key for an endpoint (plaintext encryption for private key), switch into a context with limited access to memory and narrow interface to back out to return results, and limited access to hardware, hopefully can make a stronger argument to limit the leaking of the private key across contexts. Vadim talked about some similar work/implementation/use case leveraging drivers controlling access to specific flash pages and some other interfaces.
```

File: doc/wg/core/notes/core-notes-2021-04-09.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ - Leon: Doesn't think this is a good idea. let _ is greppable. There are to groups of ignored errors: 1. errors that there's no reasonable way to handle it (e.g. inside an error handler), 2. just being lazy to handle all errors. Did some fixups to UART.
+ - Amit: Can you describe UART?
+ - Leon: e.g. error handling in the kernel, a function returns an error. Not a lot you can do. In other cases, match on the error and return the appropriate error code to the caller.
+ - Amit: Should look at places where using Result, but system is totally broken if we're returning an error.
+ - Brad: Yeah, error is meaningless at that point.
+ - Leon: Is there a way to mark those cases in the  code?
+ - Amit: A standard comment format?
+ - Leon: Yeah
+ - A let underscore with a word
+ - Hudson: \_word (or word\_ ?) has a different meaning to the compiler
```
Comment (hudson-ayers) [inline doc/wg/core/notes/core-notes-2021-04-09.md:56]:
```suggestion
 - Hudson: \_word has a different meaning to the compiler than just \_
```

**Review by lschuermann — APPROVED** [review]
Looks great, thanks!

**Review by hudson-ayers — APPROVED** [review]
Thanks for taking these!

**Comment by gemarcano** [comment]
Should I rebase and squash the extra commits into one?

**Comment by lschuermann** [comment]
@gemarcano If you ask me, I think that'd be great! However generally we tend to see GitHub PRs as the unit of change and don't usually ask people to rebase & squash explicitly.

**Review by jrvanwhy — APPROVED** [review]

**Review by ppannuto — APPROVED** [review]
bors r+

Thanks again for taking notes @gemarcano!

**Pull request merged**